src/app/ClusterInfo.h

```diff
@@ -25,96 +25,37 @@
 
 namespace chip {
 namespace app {
-static constexpr AttributeId kRootAttributeId = 0xFFFFFFFF;
 
+/**
+ * ClusterInfo is the representation of an attribute path or an event path used by ReacHandler, ReadClient, WriteHandler,
+ * Report::Engine etc, it contains a mpNext field so it can be used as a linked list. It uses some invalid values for representing
+ * the wildcard value for its field.
```

Comment:
```suggestion
 * reporting::Engine etc.  It contains a mpNext field so it can be used as a linked list.
```

It does not in fact use invalid values anymore....